src/api/lib/tasks/dev/helper_methods.rb

```diff
@@ -0,0 +1,23 @@
+def create_and_assign_project(project_name, user)
```

Comment:
We should put those methods inside a module, otherwise they are added to `Object` as private methods.